HealthMonitoring.AcceptanceTests/Properties/AssemblyInfo.cs

```diff
@@ -12,8 +12,8 @@
 [assembly: AssemblyCulture("")]
 [assembly: ComVisible(false)]
 [assembly: Guid("8d2cfa8a-f44f-4de2-98bc-b19eb496f003")]
-[assembly: AssemblyVersion("3.5.1.0")]
-[assembly: AssemblyFileVersion("3.5.1.0")]
+[assembly: AssemblyVersion("3.7.1.0")]
```

Comment:
in file environment.ps1 we have version so we dont need to update  it manually. and also we need to add record to changelog.